README.md

```diff
@@ -45,6 +46,9 @@ Clean up the cluster
 # delete the operator
 ./hack/delete-operator.sh -c $CLUSTER
 
+# If you're still using the gke cluster, you can delete persistent volumes and persistent volume claims. It is not recommended to do this in production.
+kubectl delete pv,pvc --all
```

Comment:
This can delete other existing claims or PVs which still can be dangerous. Probably, it would be enough to mention that we don't do the removal as it is the default in Kubernetes.